client/src/Picks.tsx

```diff
@@ -1026,26 +1071,44 @@ function Description({
       .then((response) => onNominationUpdate(response.body))
       .then(then)
       .catch(alertApiErrorMessage)
-      .finally(() => setEditing(false));
+      .finally(() => {
+        setEditing(false);
+        setShowHistory(false);
+      });
   };
 
   const editors = edits
     .map((edit) => edit.editor)
     .filter((u1, i, all) => u1.id !== author?.id && all.findIndex((u2) => u1.id === u2.id) === i);
 
   return editing ? (
-    <Form busyState={[busy, setBusy]} onSubmit={onSubmit}>
-      <div className='textarea-wrapper'>
-        <textarea name='description' defaultValue={text} ref={descriptionRef} />
-        <div className='description-buttons'>
-          <span>Use BBCode for formatting</span>
-          <button type='submit'>{busy ? 'Updating...' : 'Update'}</button>
-          <button type='button' onClick={() => setEditing(false)}>
-            Cancel
-          </button>
+    <>
+      <DescriptionHistory
+        author={author}
+        edits={edits}
+        showHistory={showHistory}
+        setShowHistory={setShowHistory}
+      />
+      <Form busyState={[busy, setBusy]} onSubmit={onSubmit}>
+        <div className='textarea-wrapper'>
+          <textarea name='description' defaultValue={text} ref={descriptionRef} />
+          <div className='description-buttons'>
+            <button
+              type='button'
+              disabled={edits.length === 0}
+              onClick={() => setShowHistory((prev) => !prev)}
```

Comment:
i fixed thsi already but just a "best practice" kind of thing: when setting a state based on previous value of the state, pass an action to the setter rather than conditionally choosing the value passed to setter